kong/plugins/influxdb-log/schema.lua

```diff
@@ -0,0 +1,4 @@
+local influxdb_log_schema = require "kong.plugins.http-log.schema"
+
+influxdb_log_schema.fields.content_type.default = "application/x-www-form-urlencoded"
+return influxdb_log_schema
```

Comment:
great to see the code reuse! :+1: 

just a comment on this; in Lua tables are passed by reference. So this makes the the http-log plugin and the influx-db plugin use the same table. Not just the schema, but the exact same instance of that schema.

So I think the `influxdb_log_schema` should be a deep copy of that table, before modifying it, so they are no longer using the same instance.